<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscadorPyQt</class>
 <widget class="QMainWindow" name="BuscadorPyQt">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscador-PyQt5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMenuBar" name="MenuBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>24</height>
     </rect>
    </property>
    <widget class="QMenu" name="MenuOpciones">
     <property name="title">
      <string>Opciones</string>
     </property>
     <widget class="QMenu" name="Motor_de_busqueda">
      <property name="title">
       <string>Motor de busqueda</string>
      </property>
      <addaction name="Motor_Google"/>
      <addaction name="Motor_DuckDuckGo"/>
      <addaction name="Motor_Bing"/>
     </widget>
     <widget class="QMenu" name="menuNavegador">
      <property name="title">
       <string>Navegador</string>
      </property>
      <addaction name="Nav_Google_Chrome"/>
      <addaction name="Nav_Firefox"/>
      <addaction name="Nav_Edge"/>
     </widget>
     <addaction name="Motor_de_busqueda"/>
     <addaction name="menuNavegador"/>
     <addaction name="separator"/>
     <addaction name="CloseWin"/>
    </widget>
    <addaction name="MenuOpciones"/>
   </widget>
   <widget class="QLineEdit" name="SearchVar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Ingrese el termino de busqueda:</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="StatusBar">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <action name="CloseWin">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="Motor_Google">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Google</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="Motor_DuckDuckGo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DuckDuckGo</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="Motor_Bing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bing</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="Nav_Google_Chrome">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Google Chrome</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="Nav_Firefox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Firefox</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="Nav_Edge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edge</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
